### PR TITLE
Feature/setup helpdesk

### PR DESCRIPTION
**Pull request by bimal125**
- Depends on #18

## Changes

* Add mutations to create/update/delete faqs and contact messages
* Add queries to retrieve faqs and contact messages

Mention related users here if any.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] tests
- [x] permission checks (tests here too)
- [x] translations

### CODE REVIEW

File: apps/helpdesk/filters.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    search = django_filters.CharFilter(method='search')
+
+    class Meta:
+        model = Faq
+        fields = ()
+
+    def search(self, queryset, name, value):
+        if not value:
+            return queryset
+        return queryset.filter(Q(question__icontains=value) | Q(question__icontains=value))
```
Comment (thenav56) [inline apps/helpdesk/filters.py]: why?

File: apps/helpdesk/models.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        verbose_name_plural = ugettext('Contact messages')
+
+    def __str__(self):
+        return self.message_type
+
+
+class Faq(models.Model):
+    class PublishType(models.TextChoices):
+        PUBLISH = 'publish', 'Publish'
+        DRAFT = 'draft', "Draft"
```
Comment (thenav56) [inline apps/helpdesk/models.py]:
```suggestion
        DRAFT = 'draft', 'Draft'
```

File: apps/helpdesk/mutations.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    serializer_class=FaqSerializer
+)
+
+
+class FaqMutationMixin():
+    @classmethod
+    def filter_queryset(cls, qs, info):
+        if info.context.user.user_type == User.UserType.ADMIN.value:
+            return qs
+
```
Comment (thenav56) [inline apps/helpdesk/mutations.py:30]: return qs.none()

File: apps/helpdesk/mutations.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    serializer_class=ContactMessageSerializer
+)
+
+
+class ContactMessageMutationMixin():
+    @classmethod
+    def filter_queryset(cls, qs, info):
+        if info.context.user.user_type == User.UserType.ADMIN.value:
+            return qs
+
```
Comment (thenav56) [inline apps/helpdesk/mutations.py:71]: return qs.none()

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: apps/helpdesk/filters.py (hunk 1 above)
Comment (bimal125) [reply apps/helpdesk/filters.py]: @navin typo should be question and answer

File: apps/helpdesk/models.py (hunk 2 above)
Comment (bimal125) [reply apps/helpdesk/models.py]: @thenav56  This is already changed.

**Review by thenav56 — APPROVED** [review]

**Pull request merged**